src/main/java/baritone/pathing/movement/MovementHelper.java

```diff
@@ -522,19 +548,19 @@ static boolean isBlockNormalCube(BlockState state) {
                 || block instanceof ShulkerBoxBlock) {
```

Comment:
PointedDripstoneBlock needs to be added as an exception as well.

Reply:
maybe the avoid list would be better...

Reply:
not sure what you mean, but the getCollisionShape call NullPointerExceptions on PointedDripstoneBlock

Reply:
ahh. yeah ok. I was thinking add it to movement helper line 120 as well.